nav2_behavior_tree/CMakeLists.txt

```diff
@@ -229,39 +240,59 @@ set(GENERATED_DIR ${CMAKE_CURRENT_BINARY_DIR}/gen)
 configure_file(plugins_list.hpp.in ${GENERATED_DIR}/plugins_list.hpp)
 
 add_executable(generate_nav2_tree_nodes_xml src/generate_nav2_tree_nodes_xml.cpp)
-ament_target_dependencies(generate_nav2_tree_nodes_xml ${dependencies})
+target_link_libraries(generate_nav2_tree_nodes_xml PRIVATE
+  behaviortree_cpp::behaviortree_cpp
```

Comment:
This also requires Nav2 utils as of https://github.com/ros-navigation/navigation2/pull/4492

Reply:
This is fixed in the latest push.